src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -333,6 +338,34 @@ def main():
     param_df = param_df.astype("float32")
 
     # datasub = data[['dt', 'bw', 'tw', 'twcc', 'dx', 'n', 'ncc', 'cs', 's0']]
+    
+    # STEP 4: Handle Channel Initial States
+    if showtiming:
+        start_time = time.time()
+    if verbose:
+        print("setting channel initial states ...")
+
+    if wrf_hydro_channel_restart_file:
+
+        q0 = nhd_io.get_stream_restart_from_wrf_hydro(
+            wrf_hydro_channel_restart_file,
+            wrf_hydro_channel_ID_crosswalk_file,
+            wrf_hydro_channel_ID_crosswalk_file_field_name,
+            wrf_hydro_channel_restart_upstream_flow_field_name,
+            wrf_hydro_channel_restart_downstream_flow_field_name,
+            wrf_hydro_channel_restart_depth_flow_field_name,
```

Comment:
We'll need to add commandline arguments to populate these variables (or something like them...)